[minor] added role for aws docdb provisioing

Ashish-Mahawal1 opened the pull request.

Ashish-Mahawal1 commented on ibm/mas_devops/roles/mongodb/tasks/providers/aws/provision.yml:
```diff
@@ -0,0 +1,221 @@
+    - docdb_cluster_reader_endpoint is defined and docdb_cluster_reader_endpoint != ""
+  ansible.builtin.template:
+    src: aws/docdb_tf_outputs_configmap.yml.j2
+    dest: "{{ mas_config_dir }}/docdb-cm.yml"
```
we can keep this info in MongoCfg also

Ashish-Mahawal1 replied:
done

durera reviewed (changes requested):
Needs to be merged with master and resolve the conflicts to progress

Ashish-Mahawal1 commented:
> Needs to be merged with master and resolve the conflicts to progress

@durera I have moved community dir and its contents inside providers dir and have changed filenames 
1. install.yml   <--->  provision.yml
2. uninstall.yml   <---> deprovision.yml


changed default value for ENV `MONGODB_ACTION` to provision
default for env `DB_PROVIDER` is `community`

so these changes will not produce any breaking changes
